ClientApp/KachnaOnline/src/app/shared/services/authentication.service.ts

```diff
@@ -194,4 +195,17 @@ export class AuthenticationService {
     this.user.gamificationConsent = this.kisLoggedInUserInformation.gamification_consent;
     this.user.prestige = this.kisLoggedInUserInformation.prestige;
   }
+
+  updateUserData() {
+    if (this.isLoggedIn()) {
+      if (localStorage.getItem(environment.userDataStorageName) != null) {
+        switch (localStorage.getItem(environment.userDataStorageName)) {
+          case null:
+            // TODO: Get user data.
```

Comment:
Eh, I guess this is still TODO in this PR, right?

Reply:
Yes, the whole PR is a draft and WIP. This particular method is being reworked currently.